backend/src/main/resources/templates/Rhosak/actionRequredBody.html

```diff
@@ -0,0 +1,2 @@
+{#include Rhosak/emailBodyHeader}
+<!-- TODO ->
```

Comment:
There's a typo in this file name 😃 `actionRequredBody` -> `actionRequiredBody`

Reply:
Good catch, I knew I was bound to make a mistake on the name.

Reply:
Thanks, fixed.